akka-actor/src/main/scala/akka/actor/ActorSystemSettings.scala

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright (C) 2016 Lightbend Inc. <http://www.lightbend.com>
+ */
+package akka.actor
+
+import java.util.Optional
+
+import scala.annotation.varargs
+import scala.compat.java8.OptionConverters._
+import scala.reflect.ClassTag
+
+/**
+ * Marker trait for a setting that can be put inside [[ActorSystemSettings]]
+ */
+trait ActorSystemSetting
+
+object ActorSystemSettings {
+
+  val empty = new ActorSystemSettings(Map.empty)
+
+  /**
+   * Scala API:
+   */
+  def apply(settings: ActorSystemSetting*): ActorSystemSettings =
```

Comment:
Would rather place a shortcut among the ActorSystem factories, `apply(String, ActorSystemBootstrapSettings)`